.github/workflows/tests.yml

```diff
@@ -21,7 +21,6 @@ permissions:
 
 jobs:
   tap_syntax:
-    if: github.repository == 'Homebrew/homebrew-core'
```

Comment:
Did you remove this for testing? We probably want to keep this. Same below.

Reply:
Sorry that, Wrong commit, I'm testing on my fork